ruleset.xml

```diff
@@ -1,12 +1,29 @@
 <?xml version="1.0"?>
 <ruleset name="SchererSoftware Type Hinting">
+    <arg name="extensions" value="php,ctp,js,css"/>
+    <arg name="colors"/>
+    <arg value="np"/>
+
     <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/TypeHints/DeclareStrictTypesSniff.php" />
     <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
         <properties>
             <property name="newlinesCountBetweenOpenTagAndDeclare" value="1" />
         </properties>
+        <exclude-pattern>*.ctp</exclude-pattern>
     </rule>
+
     <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/TypeHints/ReturnTypeHintSpacingSniff.php"/>
     <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/TypeHints/TypeHintDeclarationSniff.php"/>
     <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/TypeHints/LongTypeHintsSniff.php"/>
-</ruleset>
\ No newline at end of file
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/ControlStructures/YodaComparisonSniff.php"/>
+
+    <rule ref="PSR2">
+        <exclude name="PSR2.Classes.PropertyDeclaration"/>
+        <exclude name="PSR2.Methods.MethodDeclaration"/>
+        <!-- checked in SchererSoftware.SlevomatCodingStandard.ControlStructures.LenientControlSignatureSniff  -->
```

Comment:
This is not right, as the `LenientControlSignatureSniff` was removed by your hand. So this comment and the line below it references should be removed.